pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -180,7 +180,7 @@ func (c *ClusterMemberController) sync() error {
 		// }
 
 		// Pending MemberAdd: here we have observed the static pod having add dependencies filled ok to scale Cluster API.
-		if c.IsStatus("pending", p.Name, ceoapi.MemberAdd) {
+		if c.IsStatus("pending", p.Name, ceoapi.MemberReady) {
 			if err := c.etcdMemberAdd([]string{fmt.Sprintf("https://%s:2380", peerFQDN)}); err != nil {
```

Comment:
I still would like to have a distinction between a member who is in the que for addition and one that is dependency complete. Because the observation for `MemberReady` is done on the success of the init containers the Pod would technically still be `MemberUnknown` if that did not happen. I think for right now your change is fine but we might need to revisit.

Reply:
hrm makes sense, I think it would be good to something that acts as a selector, that pick one up from all the members that are dependency ready and makes it `MemberReady`.